README.md

```diff
@@ -19,6 +19,10 @@ This list includes projects under this criteria:
 
 If these conditions are met, feel free to open a PR!
 
+## Ukrainian open source maintainers in the wild
+
+* Bluesky starter pack: https://bsky.app/starter-pack/webknjaz.me/3lbt5nu3vw22b
```

Comment:
Lets made a more friendly link. Maybe something like:

* Bluesky Starter Pack: [Ukrainian FOSS maintainers and their projects](https://bsky.app/starter-pack/webknjaz.me/3lbt5nu3vw22b)